doc/lsp.txt

```diff
@@ -1411,5 +1398,65 @@ typescript-language-server for everything else: >
 		}
 	])
 <
+Another way to do it is to disable the unwanted features, for exmaple if you
+don't want diagnostics from the typescript-language-server, but want to use it
+for everything else: >
+
+	vim9script
+
+	g:LspAddServer([
+		{
+			filetype: ['javascript', 'typescript'],
+			path: '/usr/local/bin/typescript-language-server',
+			args: ['--stdio'],
+			features: {
+				diagnostics: false
+			}
+		},
+	])
+<
+==============================================================================
+17. Language Server Featured			*lsp-features*
+
+By proving the configuration |lsp-cfg-features| it's possible specify which
```

Comment:
Typo: "proving" -> "providing"
"possible specify" -> "possible to specify"

Reply:
wow it's almost impressive how many typos I managed to get in such a small change 😅